solutions/functions/prime-checker-function.js

```diff
@@ -0,0 +1,30 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+// Following function takes integer and should return True if it's prime 
+// otherwise  should return False.
+function isPrime(inputNumber){
+    // You can start below this
+    
+    if(inputNumber <= 1) return "False";
+	else if(inputNumber === 2) return "True";
+
+    for(let i = 2; i <= parseInt(Math.sqrt(inputNumber)); i++){
```

Comment:
1. Please move `parseInt(Math.sqrt(inputNumber))` outside of `for` loop. In compiled languages (like `C, C++, Java`), there is a high chance that compiler will optimize this, but I am not sure of `Javascript`, which is an interpreted language.
2. Also, please us `Math.floor`

Reply:
Addressed